server/src/lib/model/db.ts

```diff
@@ -586,4 +586,71 @@ export default class DB {
       );
     }
   }
+
+  async getUserPoints(client_id: string, challenge: string) {
+    const [[row]] = await this.mysql.query(
+      `
+      SELECT (bonus_points + clip_point + vote_point) AS user
```

Comment:
nitpick: can we pluralize all of these? ie. `bonus_points + clip_points + vote_points`

Reply:
Yeah, for consistency, surely.